standalone/src/init.ts

```diff
@@ -169,14 +166,6 @@ export default async function init(cfg: InitConfig) {
     }
     RestClient.setToken(getToken());
 
-    // initialize some basic state.
-    await Promise.all([
-        getMe()(store.dispatch, store.getState),
-        getMyPreferences()(store.dispatch, store.getState),
-        getMyTeams()(store.dispatch, store.getState),
-        getMyTeamMembers()(store.dispatch, store.getState),
-    ]);
```

Comment:
Slightly unrelated but when investigating the flow of requests I figured these were not needed for the recorder. In fact they would all fail as unauthorized since the bot wouldn't be logged in. So we are moving them on the global widget logic .